dev-db/tarantool/tarantool-9999.ebuild

```diff
@@ -41,7 +41,7 @@ HOMEPAGE="https://tarantool.org"
 IUSE="
 	+backtrace debug feedback-daemon gcov gprof +system-libcurl
 	+system-libyaml +system-zstd systemd test cpu_flags_x86_sse2
-	cpu_flags_x86_avx
+	cpu_flags_x86_avx embed-luarocks
```

Comment:
Nit: It seems, the flags are sorted alphabetically here (except `CPU_FLAGS_X86`, which are at the end).

Reply:
FIxed